core/debug/src/debug_box.c

```diff
@@ -142,3 +142,46 @@ void debug_register_driver(const TUvisorDebugDriver * const driver)
     g_debug_box.box_id = g_active_box;
     g_debug_box.initialized = 1;
 }
+
+/* XXX This is a bit platform specific. Consider moving to an platform specific
```

Comment:
Yes, will move to a FIXME. We'll refactor when we have another architecture to add support for.